ibis/impala/metadata.py

```diff
@@ -43,15 +43,15 @@ def _converter(tup):
 
 def _try_timestamp(x):
     try:
-        ts = pd.Timestamp(x)
-        return ts.to_pydatetime()
+        ts = pd.Timestamp(x, tz='UTC')
+        return ts.to_pydatetime().replace(tzinfo=None)
     except (ValueError, TypeError):
         return x
 
 
 def _try_unix_timestamp(x):
     try:
-        ts = pd.Timestamp.fromtimestamp(int(x))
+        ts = pd.Timestamp.utcfromtimestamp(int(x))
```

Comment:
@wesm Just want to call your attention to these changes. This ensures that timestamps look the same no matter what timezone Python is run in. I actually only discovered this when I was in NY a while ago.